TA_Scheduler/views/account/create.py

```diff
@@ -1,13 +1,29 @@
 from django.shortcuts import render
 from django.views import View
-from TA_Scheduler.models import Account
 from TA_Scheduler.utilities.AccountUtil import AccountUtil
 
 
 class CreateAccount(View):
+
+    """Called when the user opens the Create Account page.
+
+    :param request: request from account/create.html
+    :return: rendering of account/create.html with blank form
+    :pre: User is not anonymous, instructor, or ta
+    :post: None
+    """
```

Comment:
The docs look great. it just needs to go under the function instead of above it.